src/glk/gridmap.cpp

```diff
@@ -40,13 +39,13 @@ GridMap::GridMap(double resolution, int width, int height, const unsigned char*
   init_vao(resolution, width, height);
 }
 
-GridMap::GridMap(double resolution, int width, int height, float scale, const float* values, float alpha, ColorMode mode) {
+GridMap::GridMap(double resolution, int width, int height, float scale, const float* values, float alpha, ColorMode mode) : mode(mode) {
   std::vector<float> rgba(width * height * 4);
```

Comment:
Same as the `unsigned char` constructor. The initialization can be replaced with a single call of `update_color()`.